src/js/components/Form/__tests__/__snapshots__/Form-test.js.snap

```diff
@@ -277,7 +270,6 @@ exports[`Form with field 1`] = `
   box-sizing: border-box;
   outline: none;
   max-width: 100%;
-  margin-bottom: 12px;
```

Comment:
do we know why this snapshot changed? it looks like a potential ui regression.

Reply:
As far as I know, when I added it **margin** as a prop, and tried a test, it showed this change. This was before the update of the snapshot, I can try to find what's causing this change. 
Thanks for the review and feedback @ShimiSun!

Reply:
@ShimiSun , if I remove the *margin* from the **props** object and pass **{...formField.margin}** instead, *margin-top,margin-bottom,margin-right,margin-left* are being added to my snapshot when running **yarn test**.

![Screenshot from 2019-05-31 14-33-27](https://user-images.githubusercontent.com/35984969/58694624-136ef680-83b1-11e9-9679-d6c9c545ec06.png)